java-common/src/main/java/com/typesafe/conductr/java/Tuple.java

```diff
@@ -0,0 +1,54 @@
+package com.typesafe.conductr.java;
+
+/**
+ * Play's Tuple class given Java 8's lack of it.
+ */
+public class Tuple<A, B> {
+    public final A _1;
+    public final B _2;
+
+    public Tuple(A var1, B var2) {
+        this._1 = var1;
+        this._2 = var2;
+    }
+
+    public String toString() {
+        return "Tuple2(_1: " + this._1 + ", _2: " + this._2 + ")";
```

Comment:
What about:

```java
 public String toString() {
    return "(" + this._1: + ", " + this._2 + ")";
}
```

This would be equivalent to Scala.